conf/docker-aio/testdata/httpd.conf

```diff
@@ -0,0 +1,30 @@
+
+Include conf.d/*.conf
+Include conf.modules.d/*.conf
+ServerName localhost
+Listen 80 443
+PidFile run/httpd.pid
+DocumentRoot "/var/www/html"
+TypesConfig /etc/mime.types
+User apache
+Group apache
+
+<VirtualHost *:80>
+  ServerName localhost
+  LogLevel debug
+ ErrorLog logs/error_log
+ LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+ CustomLog logs/access_log combined
+ 
+  # proxy config (aka - what to send to glassfish or not)
+  ProxyPassMatch ^/RApacheInfo$ !
+  ProxyPassMatch ^/custom !
+  ProxyPassMatch ^/dataexplore !
+  ProxyPassMatch ^/Shibboleth.sso !
+  ProxyPassMatch ^/shibboleth-ds !
+  # pass everything else to Glassfish
+  ProxyPass / ajp://localhost:8009/
+# glassfish can be slow sometimes
+  ProxyTimeout 300 
+
+</VirtualHost>
```

Comment:
What is this file for?

Reply:
Its a file that was missing from the docker-aio folder. .gitignore has blocked it when the configuration was last committed.

Reply:
Oh. Ok. Thanks for explaining this.